packages/integrations/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@homarr/integrations",
+  "private": true,
+  "version": "0.1.0",
+  "exports": {
+    ".": "./index.ts",
+    "./types": "./src/types.ts"
+  },
+  "typesVersions": {
+    "*": {
+      "*": [
+        "src/*"
+      ]
+    }
+  },
+  "license": "MIT",
```

Comment:
Are we sure about this license ? 
In general I think we discussed about having a license that does not allow commercial use without a written agreement

Reply:
Out of scope - let's discuss this via chat

Reply:
Then we would no longer be open source in the general sense and only source-available